config/code_schools.yml

```diff
@@ -735,3 +735,19 @@
       city: Seattle
       state: WA
       zip: 98121
+
+- name: Code Platoon
+  url: https://www.codeplatoon.org
+  logo: https://raw.githubusercontent.com/OperationCode/operationcode_frontend/code-schools-cooper-kyle/src/images/codeSchoolLogos/code_platoon.jpg
```

Comment:
Can you change this to 
`https://raw.githubusercontent.com/OperationCode/operationcode_frontend/master/src/images/codeSchoolLogos/code_platoon.jpg` ?